srcpkgs/evolution/template

```diff
@@ -1,25 +1,25 @@
 # Template file for 'evolution'
 pkgname=evolution
-version=3.34.1
-revision=4
+version=3.34.2
+revision=1
 build_style=cmake
 configure_args="-DENABLE_AUTOAR=OFF -DENABLE_LIBCRYPTUI=OFF -DENABLE_GTKSPELL=OFF
  -DENABLE_TEXT_HIGHLIGHT=OFF -DENABLE_PST_IMPORT=OFF
  -DENABLE_WEATHER=$(vopt_if gir ON OFF)"
 hostmakedepends="gnome-doc-utils gobject-introspection intltool itstool
  pkg-config"
 makedepends="NetworkManager-devel clutter-gtk-devel enchant2-devel
- evolution-data-server-devel gspell-devel gtkhtml-devel libcanberra-devel
- $(vopt_if gir libgweather-devel) libnotify-devel webkit2gtk-devel ytnef-devel
- gnome-desktop-devel"
+ evolution-data-server-devel>=$version gspell-devel gtkhtml-devel
```

Comment:
Don't use version constraints in makedepends, the build system of the package you are building should be the one checking it. Also it is breaking quite a few assumptions in `xbps-src` (e.g. breaks `show-build-deps`). They should only be used for runtime dependencies.

Reply:
Ok. I will not do this anymore.